src/cordova/plugin/android/java/build-extras.gradle

```diff
@@ -5,8 +5,8 @@ repositories {
 }
 
 dependencies {
+  implementation 'org.outline:outline:0.0'
```

Comment:
I'm wondering if we can clean up the command line, though not needed for this PR if it's already working.

Does `fileTree` work here? 
Alternatively, does the `includeBuild` setting works here?

In either case you probably need to assume the project being built from inside the `/platforms/android/` directory.

Reply:
I don't know of a way to use `fileTree` because there isn't a specific file or pre-built artifact that we are trying to include.  `includeBuild` can only be used in `settings.gradle`, which is auto-generated and [cannot be customized](https://stackoverflow.com/questions/34307879/is-it-possible-to-manually-edit-settings-gradle-file), so I don't see a way to use that either.

It may be possible to get rid of the command-line argument by creating something like a local maven repo, but Gradle [recommends against this](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:case-for-maven-local).

The `--include-build` solution may not be the most elegant, but I do like the result that this line looks like an ordinary dependency import.  In principle, we could move the library to a different repo, and publish it to Maven Central, and this would still work.